### ISSUE TITLE
[Suggestion]: More Modifier Key Options

### ISSUE THREAD

**Issue by Zombo2112** [Fuzss/slotcycler#17]
### Mod Loader (Optional)

Fabric

### Minecraft Version(s) (Optional)

1.20.4

### Suggestion (Required)

Options to extend the functionality of the "scrolling modifier key" to the other forms of swapping:

-      Double press of a hotbar key swaps items only while Modifier key is held.

-      Cycle Keybinds Left/Right only swap items while Modifier key is held. 

(For example if someone wants to set their keybinds to 1 and 3 to Cycle Left and Cycle Right respectively, they dont end up swapping with their inventory every time they press 1 to go back to their first hotbar slot.)

**Comment by nothere333** [comment]
Yes, it should be a keybind, why limit the options?

**Comment by Hyperbuss** [comment]
Agreed, alt is hot real-estate for modded players for things like Litematica, Create, and Tweakeroo.
